mdoc/src/main/scala-2/mdoc/internal/markdown/Renderer.scala

```diff
@@ -92,6 +93,23 @@ object Renderer {
     }
   }
 
+  def appendMultilineMaxWidth(sb: PrintStream, string: String, maxLineLength: Int): Unit = {
```

Comment:
I'm used to thinking about LINE:COLUMN so I originally interpreted "maxLineLength" as "maxHeightLength"

```suggestion
  def appendMultilineMaxWidth(sb: PrintStream, string: String, maxWidth: Int): Unit = {
```